src/index.ts

```diff
@@ -74,7 +76,9 @@ function getWeb3SwapClient(database: Database) {
   const address: string = process.env.SWAP as string;

```

Comment:
You can pull in `swap-erc20` addresses from `@airswap/swap-erc20/deploys.js` FYI.